py/.gitignore

```diff
@@ -0,0 +1,7 @@
+sentry_data_schemas/event.schema.json
+
+# Wheels
+dist
```

Comment:
Do we really want nested `.gitignore` files?

Reply:
My mistake, sorry, I moved this around and forgot to remove it